webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,101 @@
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
-import Skeleton from 'react-loading-skeleton';
-import { ArrowIcon } from '@patternfly/react-icons';
-import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import React from 'react';
 import {
+  Bullseye,
   Card,
   DataList,
   DataListItem,
   DataListItemRow,
   DataListItemCells,
+  DataListWrapModifier,
   DataListCell,
+  CardActions,
   CardBody,
   CardTitle,
-  Accordion,
-  AccordionItem,
-  AccordionContent,
-  AccordionToggle,
+  CardHeader,
+  Title,
 } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
 
 import { foremanUrl } from '../../../../foreman_tools';
```

Comment:
You can import it directly from
```suggestion
import { foremanUrl } from '../../../common/helpers';
```
(tbh i'm not sure why it isn't in urlHelpers, but since we might be dropping it i wouldn't bother moving it right now)